### PR TITLE
CTO-71 - Added upstream/downstream location endpoints and controllers…

### PR DESCRIPTION
**Pull request by rma-bryson**
…. Fixed issue with DTOs needing JSONignore tags

### CODE REVIEW

File: cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/StreamLocationNode.java [hunk 1]
```diff
@@ -68,7 +69,7 @@ public void setStreamNode(StreamNode streamNode) {
     }
 
     //helper methods to extract from stream node
-
+    @JsonIgnore
```
Comment (adamkorynta) [inline cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/StreamLocationNode.java:72]: Why are these needed?

Comment (rma-bryson) [reply cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/StreamLocationNode.java:72]: They were just to helpers to reduce code on calls

streamLoc.getStreamLocationNode().getStreamNode().getStreamId()

vs

streamLoc.getStreamId()

Comment (rma-bryson) [reply cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/StreamLocationNode.java:72]: Right. Added these onto the codegen

**Review by adamkorynta — APPROVED** [review]

**Review by adamkorynta — APPROVED** [review]
